run.py

```diff
@@ -217,6 +217,11 @@ def train(
             init=init.Normal(sigma=0.1),
         )
 
+    for key, value in model.collect_params().items():
+        data = value.data()
+        print(f'{key!r} weights norm: {np.linalg.norm(data).item():.3f}')
+        print(f'{key!r} grad norm: {np.linalg.norm(data.grad).item():.3f}')
+
```

Comment:
Debug messages should be optional; either remove or add a verbosity level check.